packages/perseus-editor/src/__stories__/content-preview.stories.tsx

```diff
@@ -4,7 +4,7 @@ import {
 } from "@khanacademy/perseus";
 import {View} from "@khanacademy/wonder-blocks-core";
 import {spacing} from "@khanacademy/wonder-blocks-tokens";
-import {useState} from "react";
+import * as React from "react";
```

Comment:
Not importing React was violating the `react/react-in-jsx-scope` rule. React 18 apparently uses the new JSX runtime which doesn't require this, but I'm not 100% sure if we are using the new JSX runtime (we are in Storybook/Vite, but not sure about in our production builds). I'm going to add this here for now instead of disabling the rule.